CMakeLists.txt

```diff
@@ -180,20 +180,13 @@ add_subdirectory(samples)
 option(BUILD_TESTS "Enable tests compilation" OFF)
 mark_as_advanced(BUILD_TESTS)
 
-option(BUILD_ASE_TESTS "Enable ASE test compilation." OFF)
-mark_as_advanced(BUILD_ASE_TESTS)
-
 option(ENABLE_MOCK "Enable mock driver for testing." ON)
 mark_as_advanced(ENABLE_MOCK)
 
-if(BUILD_TESTS OR BUILD_ASE_TESTS)
```

Comment:
You removed BUILD_ASE_TESTS option. Does this mean ASE tests will always be built whenever BUILD_TESTS is ON? Should we check if BUILD_ASE is ON before building ASE tests?

Reply:
BUILD_ASE_TESTS option was to build ASE gtest from tests folder. Actually I am going to remove the tests folder from OPAE repo as well. You can build ASE tests with build target test_ase in the updated scripts/cover.sh.